setup.py

```diff
@@ -244,32 +256,27 @@ def extract_version():
 }
 
 
-pypi_name = "scitools-iris"
-
-with open(os.path.join(SETUP_DIR, "README.md"), "r") as fh:
-    description = "".join(fh.readlines())
-
 setup(
-    name=pypi_name,
+    name=PYPI_NAME,
     version=extract_version(),
     url="http://scitools.org.uk/iris/",
     author="UK Met Office",
     author_email="scitools-iris-dev@googlegroups.com",
     description="A powerful, format-agnostic, community-driven Python "
     "library for analysing and visualising Earth science data",
-    long_description=description,
+    long_description=long_description(),
     long_description_content_type="text/markdown",
     packages=find_package_tree("lib/iris", "iris"),
     package_dir={"": "lib"},
     include_package_data=True,
     cmdclass=custom_commands,
     zip_safe=False,
     setup_requires=pip_requirements("setup"),
-    install_requires=pip_requirements("setup") + pip_requirements("core"),
-    tests_require=["{}[test]".format(pypi_name)],
+    install_requires=pip_requirements("setup", "core"),
+    tests_require=[f"{PYPI_NAME}[test]"],
     extras_require={
-        "test": pip_requirements("test"),
         "all": pip_requirements("all"),
-        "extensions": pip_requirements("extensions"),
```

Comment:
Dropping the use of `extensions` as `cartopy` will now pull in `gdal`, and `iris-grib` is not used now with the `conda` requirements, so mirroring that here... hence, no `extensions`.